docs/contributor_guide/developer_call_notes/2020_11_12.rst

```diff
@@ -0,0 +1,58 @@
+Developer Call 2020-10-08
```

Comment:
Please update the date. Also, the previous one didn't have the same format as all previous ones (which spelled out the month). Please unify.

Reply:
Fixed the date; is it worth converting all the others to ISO format?

Reply:
It's all manually written. There's no automatic conversion. That's why we should change this and the previous one as well:

![image](https://user-images.githubusercontent.com/10245648/99116795-7ce4c400-25a9-11eb-91da-5c73a8b77592.png)

Reply:
@riedgar-ms  I think you wrote "<n>th <month>" now as opposed to "<month> <n>". I'm fine either way, but let's be consistent with all of them (including last month's). So you can either make all of them conform to your way of writing it or adjust the last two to the existing way.

Reply:
I'm afraid the ordering is still different. Yours says 8 October 2020, for example, whereas the ones above (in the screenshot) say September 17, 2020. Please just get them all aligned in whichever way you prefer.